### PR TITLE
[Maintenance] Enable some tests

### PR DESCRIPTION
**Pull request by GSadee**

### CODE REVIEW

File: src/Controller/Checkout/ShowAvailablePaymentMethodsAction.php [hunk 1]
```diff
@@ -29,25 +32,33 @@ final class ShowAvailablePaymentMethodsAction
...
     }
 
     public function __invoke(Request $request): Response
     {
         /** @var OrderInterface $cart */
         $cart = $this->cartRepository->findOneBy(['tokenValue' => $request->attributes->get('token')]);
 
-        $payments = [];
+        if (!$this->isCheckoutTransitionPossible($cart, OrderCheckoutTransitions::TRANSITION_SELECT_PAYMENT)) {
+            throw new BadRequestHttpException('The payment methods cannot be resolved in the current state of cart!');
```
Comment (mamazu) [inline src/Controller/Checkout/ShowAvailablePaymentMethodsAction.php]: I would suggest to use a different exception to not tie this logic to the HTTP - Layer. Isn't there a general transition not possible exception?

Comment (lchrusciel) [reply src/Controller/Checkout/ShowAvailablePaymentMethodsAction.php]: We are in the controller, so coupling with HTTP is totally ok. We could, however, add a more generic exception like `Transition not possible`, but this one is fair enough for me.

Comment (mamazu) [reply src/Controller/Checkout/ShowAvailablePaymentMethodsAction.php]: Ah okay, didn't notice that.

**Comment by lchrusciel** [comment]
Thank you, Grzegorz! :tada:

**Pull request merged**